state/protocol/kvstore.go

```diff
@@ -52,7 +52,7 @@ type KVStoreReader interface {
 	// different extensions can have different view counts.
 	GetEpochExtensionViewCount() uint64
 
-	// GetEpochCommitSafetyThreshold [t] defines a deadline for sealing the EpochCommit
+	// GetFinalizationSafetyThreshold [t] defines a deadline for sealing the EpochCommit
```

Comment:
@AlexHentschel can you I ask you update following comment which explains what is a finalization safety threshold as protocol parameter which is used in several places and not only epoch commitment?

Reply:
I have updated the documentation of `FinalizationSafetyThreshold` :point_right:  my [PR 6384](https://github.com/onflow/flow-go/pull/6384).